src/App.test.tsx

```diff
@@ -54,16 +54,6 @@ describe("policy list item", () => {
     expect(within(firstItem).getByText(/AZ78YUV/i)).toBeVisible();
   });
 
-  test("customer name", async () => {
```

Comment:
do we want to remove this?

Reply:
As discussed the customer name is no longer on the list item card - I've added a dialog specific customer test below